File: 119.pascals-triangle-ii.cpp
class Solution {
public:
    vector<int> getRow(int rowIndex) {
        vector<int> row;
        for (int i = 0; i <= rowIndex; i++) {
            row.push_back(1);
            for (int j = row.size() - 2; j > 0; j--) {
                row[j] = row[j - 1] + row[j];
            }
        }
        return row;
    }
};

File: 119.pascals-triangle-ii.py
class Solution:
    def getRow(self, rowIndex: 'int') -> 'List[int]':
        row = []
        for i in range(rowIndex + 1):
            row.append(1)
            for j in range(len(row) - 2, 0, -1):
                row[j] = row[j - 1] + row[j]
        return row
